client/my-sites/sharing/style.scss

```diff
@@ -895,29 +895,29 @@
 		background: $color;
 
 		&:hover {
-			background: rgba( $color, 0.9 );
+			background: $color;
 		}
 	}
 }
 
-@include sharing-button-service( 'facebook', $color-facebook );
-@include sharing-button-service( 'twitter', $color-twitter );
-@include sharing-button-service( 'press-this', $blue-wordpress );
-@include sharing-button-service( 'path', $color-path );
-@include sharing-button-service( 'google-plus-1', $color-gplus );
-@include sharing-button-service( 'instagram', $color-instagram );
-@include sharing-button-service( 'eventbrite', $color-eventbrite );
-@include sharing-button-service( 'linkedin', $color-linkedin );
-@include sharing-button-service( 'stumbleupon', $color-stumbleupon );
-@include sharing-button-service( 'tumblr', $color-tumblr );
-@include sharing-button-service( 'reddit', $color-reddit );
-@include sharing-button-service( 'pinterest', $color-pinterest );
-@include sharing-button-service( 'pocket', $color-pocket );
-@include sharing-button-service( 'email', $color-email );
-@include sharing-button-service( 'print', $color-print );
-@include sharing-button-service( 'skype', $color-skype );
-@include sharing-button-service( 'telegram', $color-telegram );
-@include sharing-button-service( 'jetpack-whatsapp', $color-whatsapp );
+@include sharing-button-service( 'facebook', var( --color-facebook ));
+@include sharing-button-service( 'twitter', var( --color-twitter ));
+@include sharing-button-service( 'press-this', var( --color-wpcom ));
+@include sharing-button-service( 'path', var( --color-path-rgb ));
```

Comment:
Seems to be an rgb here?

Reply:
Oops, removed.

Sidenote: I've not checked if it's used across Calypso yet, but does Path even need to be included at all? It closed down a while ago.

Reply:
The color is still being used here https://github.com/Automattic/wp-calypso/blob/006d96c71bf6e41f070bed44ddb655f634b9c52b/client/blocks/post-share/style.scss#L241

But I don't think it's an option anymore, so probably fine to remove

![screen shot 2019-01-17 at 1 38 55 pm](https://user-images.githubusercontent.com/618551/51344081-7a70bc00-1a5d-11e9-8f09-b290aa5fee4e.png)